zerver/openapi/zulip.yaml

```diff
@@ -156,8 +156,39 @@ paths:
                           guaranteed to be increasing, but they are not guaranteed to be
                           consecutive.
                         items:
-                          type: object
-                          additionalProperties: false
+                          oneOf:
+                          - type: object
```

Comment:
I think this isn't documenting the `type: string` that every event has?  I feel like we want that to be a common include.

Reply:
Yes I will make a $ref for this and various other things as there are quite a lot of repetitions.